lib/manager/cocoapods/readme.md

```diff
@@ -0,0 +1,232 @@
+
+To update all packages: `pod update`
+
+---
+
+#### If applicable, describe how the tool maintains a cache and if it can be controlled via CLI or env? Do you recommend the cache be kept or disabled/ignored?
+
+Cocoapods keeps cache in directory `${HOME}/Library/Caches/CocoaPods`
+
+You can get information about cache state and clean cache throutgh cocoapods ([details](https://guides.cocoapods.org/terminal/commands.html#group_cache))
+
+---
+
+#### If applicable, what command should be used to generate a lock file from scratch if you already have a package file? This will be used for "lock file maintenance".
+
+You can generate `Podfile.lock` by running `pod install`. This command also install required dependencies and link them to xcode project.
+
+I don't know hot to generate only `Podfile.lock`
+
+## Other
+
+#### Is there anything else to know about this package manager?
+
+Cocoapods has interesting feature that may be usefull for renovate.
+Command `pod outdated` lists all outdated dependecies ([details](https://guides.cocoapods.org/using/pod-install-vs-update.html#pod-outdated))
+
+---
+
+In order to add dependencies to your xcode project you have to create file named `Podfile`
+in your Xcode project directory
+
+```
+platform :ios, '8.0'
+use_frameworks!
+
+target 'MyApp' do
+  pod 'AFNetworking', '~> 2.6'
+  pod 'ORStackView', '~> 3.0'
+  pod 'SwiftyJSON', '~> 2.3'
+end
+```
+
+After that you can install dependencies in your project by runing `pod install`
+
+Also pod CLI supports creating own pod `pod spec create Peanut`
+
+This command will create a podspec file. This file describes a version of Pod library. It includes details about where the source should be fetched from, what files to use, the build settings to apply, and other general metadata such as its name, version, and description. More details about podspec syntax you can find [here](https://guides.cocoapods.org/syntax/podspec.html)
+
+```
+Pod::Spec.new do |spec|
+  spec.name         = 'Reachability'
+  spec.version      = '3.1.0'
+  spec.license      = { :type => 'BSD' }
+  spec.homepage     = 'https://github.com/tonymillion/Reachability'
+  spec.authors      = { 'Tony Million' => 'tonymillion@gmail.com' }
+  spec.summary      = 'ARC and GCD Compatible Reachability Class for iOS and OS X.'
+  spec.source       = { :git => 'https://github.com/tonymillion/Reachability.git', :tag => 'v3.1.0' }
+  spec.source_files = 'Reachability.{h,m}'
+  spec.framework    = 'SystemConfiguration'
+  spec.dependency 'SomeOtherPod'
+end
+```
+
+When you're done you can get an account and push your pod to the CocoaPods Trunk. CocoaPods Trunk is an authentication and CocoaPods API service. To publish new or updated libraries to CocoaPods for public release you will need to be registered with Trunk and have a valid Trunk session on your current device. More details about CocoaPods Trunk you can find [here](https://guides.cocoapods.org/making/getting-setup-with-trunk.html)
```

Comment:
So is this how you publish packages to the main repo? i.e. not via Pull Requests, but through this service?

Reply:
Yes, exactly. You have to use this tool in order to publish your pod.